lib/hanami/events/adapter/memory_async.rb

```diff
@@ -35,6 +37,10 @@ def thread_spawned!
           @thread_spawned = true
         end
 
+        def id_generator
+          SecureRandom.hex(8)
```

Comment:
I'm not sure that it should be uuid. now it's just random id without any mask or something like this. We can research this topic later and find the best way to generate id